io.github.maddecoder.Classic-RBDOOM-3-BFG.yml

```diff
@@ -0,0 +1,139 @@
+app-id: io.github.maddecoder.Classic-RBDOOM-3-BFG
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "22.08"
+command: DOOMBFA.sh
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+build-options:
+    append-path: /usr/lib/sdk/mono6/bin
+    append-ld-library-path: /usr/lib/sdk/mono6/lib
+    append-pkg-config-path: /usr/lib/sdk/mono6/lib/pkgconfig
+
+finish-args:
+- --device=all
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --share=network
+- --socket=pulseaudio
+- --filesystem=~/.var/app:ro
+- --filesystem=home
+- --filesystem=/run/media:ro
+- --filesystem=/media:ro
+
+cleanup:
+- /include
+- /lib/*.a
+- /lib/*.la
+- /lib/pkgconfig
+- /lib/cmake
+- /lib/debug
+- /share/man
+- /share/doc
+
+modules:
+- libgdi.yml
+- name: DoomBFA
+  buildsystem: cmake
```

Comment:
cmake-ninja?

Reply:
Does it need any additional configuration from cmake?

Reply:
`buildsystem: cmake-ninja`

as per the documentation. it's prefered because it's faster.

Reply:
done, both the main project and OpenAL-soft